docs/usage/gardener-api.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Accessing the Gardener API
+---
+
+# Accessing the Gardener API Through the Dashboard
+
+## Overview
+
+The cluster operations that are performed manually in the dashboard or via `kubectl` can be automated using the [**Gardener API**](https://github.com/gardener/gardener/blob/master/docs/api-reference/README.md) and a **Service Account** authorized to perform them.
+
+> Note: A project **Service Account** has access to all Kubernetes resources in the project. Treat with care.
```

Comment:
it depends on the given permission ;)

Reply:
Would the following correction work? 

"Note: Depending on the given permission, a project **Service Account** could have access to all Kubernetes resources in the project. Treat with care."

Reply:
I would even remove the complete note. Not sure how much value it adds. 

Instead the downloaded kubeconfig (described at the end of the document) should be treated with care as it contains credentials. Maybe this is a more meaningful hint